src/common/cluster-store.ts

```diff
@@ -137,31 +139,27 @@ export class ClusterStore extends BaseStore<ClusterStoreModel> {
     });
 
     makeObservable(this);
-
+    this.load();
     this.pushStateToViewsAutomatically();
   }
 
-  async load() {
-    const initialStates = "cluster:states";
-
-    await super.load();
-
-    if (ipcRenderer) {
-      logger.info("[CLUSTER-STORE] requesting initial state sync");
+  async loadInitialOnRenderer() {
```

Comment:
This had to be moved to its own function since `load()` is no longer `async`. I still believe that this is a better design.